src/terra-dev-site/example/ResponsiveListAndCellGridExample.jsx

```diff
@@ -0,0 +1,174 @@
+    </CellGrid>
+    <CellGrid className={cx('row')}>
+      <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }} />
+      <Cell className={cx('display')} key="cell-1" width={{ scalar: 1 }}>{display1}</Cell>
+      <Cell className={cx('display')} key="cell-2" width={{ scalar: 1 }}>{display2}</Cell>
+      <Cell className={cx('display')} key="cell-3" width={{ scalar: 1 }}>{display3}</Cell>
+      <Cell className={cx('display')} key="cell-4" width={{ scalar: 1 }}>{display4}</Cell>
+      <Cell className={cx('display')} key="cell-5" width={{ scalar: 1 }}>{display5}</Cell>
+      <Cell className={cx('display')} key="cell-6" width={{ scalar: 1 }}>{display6}</Cell>
+      <Cell className={cx('comment')} key="cell-7" width={{ scalar: 1 }}>{comment}</Cell>
+      <Cell className={cx('end-accessory')} key="cell-8" width={{ static: { value: 40, unit: 'px' } }}>{endAccessory}</Cell>
+    </CellGrid>
+    <CellGrid className={cx('row')}>
+      <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }}>{startAccessory}</Cell>
+      <Cell className={cx('display')} key="cell-1" width={{ scalar: 1 }}>{display1}</Cell>
+      <Cell className={cx('display')} key="cell-2" width={{ scalar: 1 }}>{display2}</Cell>
+      <Cell className={cx('display')} key="cell-3" width={{ scalar: 1 }}><ItemView.Display text="Room 100A" textStyle="strikeThrough" /></Cell>
+      <Cell className={cx('display')} key="cell-4" width={{ scalar: 1 }}>{display4}</Cell>
+      <Cell className={cx('display')} key="cell-5" width={{ scalar: 1 }}>{display5}</Cell>
+      <Cell className={cx('display')} key="cell-6" width={{ scalar: 1 }}>{display6}</Cell>
+      <Cell className={cx('comment')} key="cell-7" width={{ scalar: 1 }}>{comment}</Cell>
+      <Cell className={cx('end-accessory')} key="cell-8" width={{ static: { value: 40, unit: 'px' } }}>{endAccessory}</Cell>
+    </CellGrid>
+    <CellGrid className={cx('row')}>
+      <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }}>{startAccessory}</Cell>
+      <Cell className={cx('display')} key="cell-1" width={{ scalar: 1 }}>{display1}</Cell>
+      <Cell className={cx('display')} key="cell-2" width={{ scalar: 1 }}>{display2}</Cell>
+      <Cell className={cx('display')} key="cell-3" width={{ scalar: 1 }}>{display3}</Cell>
+      <Cell className={cx('display')} key="cell-4" width={{ scalar: 1 }}>{display4}</Cell>
+      <Cell className={cx('display')} key="cell-5" width={{ scalar: 1 }}>{display5}</Cell>
+      <Cell className={cx('display')} key="cell-6" width={{ scalar: 1 }}>{display6}</Cell>
+      <Cell className={cx('comment')} key="cell-7" width={{ scalar: 1 }}>{comment}</Cell>
+      <Cell className={cx('end-accessory')} key="cell-8" width={{ static: { value: 40, unit: 'px' } }} />
+    </CellGrid>
+  </div>
+);
+
+const CustomItemCollection = () => (
+  <ResponsiveElement
+    responsiveTo="parent"
```

Comment:
The props tiny and small are deprecated. This should use onChange instead to manage flipping these instead.

Another question, should orion-application breakpoints package be used instead? Responsive element can cause a flicker on initial load

Reply:
Swapped to breakpoints https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40#diff-5bb9914536b23dafae1ea592488ce19fda98bedad1874ac8546a1dedbc6cdd38R188